wiki/Tournaments/GTS/CGTS_2021/en.md

```diff
@@ -0,0 +1,587 @@
+6. All players must stay respectful and keep a proper attitude. Not following this rule can result in a ban/blacklist from the tournament. This rule concerns all the staff as well.
+7. Any rule changes or unexpected occurrences will be announced in the thread and in the Discord server.
+
+### Restrictions and registrations
+
+1. Players must join the Discord server of the tournament to actually be counted as a player. If not, then they won't get on any player lists.
+2. Players sign up individually. When their registration is accepted, they will get a player role with their respective region.
+3. The captain for each region's team by default will be the winner of the 1v1 tournament for that region. He can either ask to change captain, if him and the new captain agree, or it can be changed by the team, if everyone else on the team agrees.
+4. The captain is going to be the one who reschedules a match if needed and will ensure that his teammates are here for the matches.
+5. Followings are the list of countries that can play from different continents:
+   - Asia: Afghanistan, Armenia, Azerbaijan, Bahrain, Bangladesh, Bhutan, China, Cyprus, Georgia, Hong Kong, India, Iran, Iraq, Israel, Japan, Jordan, Kazakhstan, Kuwait, Kyrgyzstan, Lebanon, Maldives, Mongolia, Nepal, Oman, Pakistan, Palestine, Qatar, Saudi Arabia, South Korea, Sri Lanka, Syria, Taiwan, Tajikistan, Turkmenistan, United Arab Emirates (UAE), Uzbekistan, Yemen.
+   - Europe: Andorra, Austria, Belgium, Czech Republic, Denmark, Estonia, Finland, France, Germany, Greece, Iceland, Italy, Ireland, Latvia, Liechtenstein, Lithuania, Luxembourg, Moldova, Monaco, Norway, Netherlands, Poland, Portugal, Romania, Russian Federation, Spain, Sweden, Switzerland, Turkey, United Kingdom, Ukraine.
+   - North America: Bahamas, Barbados, Belize, Canada, Costa Rica, Cuba, Dominica, Dominican Republic, El Salvador, Grenada, Guatemala, Haiti, Honduras, Jamaica, Mexico, Nicaragua, Panama, Saint Kitts and Nevis, Saint Lucia, Saint Vincent and the Grenadines, Trinidad and Tobago, United States of America.
+   - Oceania/South East Asia: Australia, Brunei, Cambodia, Fiji, Indonesia, Laos, Malaysia, Marshall Islands, Micronesia, Myanmar, Nauru, New Zealand, Palau, Papua New Guinea, Philippines, Samoa, Singapore, Solomon Islands, Thailand, Tonga, Tuvalu, Vanuatu, Vietnam.
+   - South America: Argentina, Bolivia, Brazil, Chile, Colombia, Ecuador, Guyana, Paraguay, Peru, Suriname, Uruguay, Venezuela.
+
+### Stage regulations
+
+1. There are seven stages in the 1v1 tournament, the Qualifiers, the RO32, the RO16, the Quarterfinals, the Semifinals, the Finals, and the Grand Finals. It may change depending on if a region gets less players than expected, or much more than expected.
+2. There are five stages in the team tournament, the Qualifiers, the Quarterfinals, the Semifinals, the Finals, and the Grand Finals.
```

Comment:
```suggestion
1. There are seven stages in the 1v1 tournament: the Qualifiers, the RO32, the RO16, the Quarterfinals, the Semifinals, the Finals, and the Grand Finals. It may change depending on if a region gets less players than expected, or much more than expected.
2. There are five stages in the team tournament: the Qualifiers, the Quarterfinals, the Semifinals, the Finals, and the Grand Finals.
```